pathd/path_pcep_cli.c

```diff
@@ -526,7 +528,7 @@ static int path_pcep_cli_show_srte_pcep_counters(struct vty *vty)
 	strftime(tm_buffer, sizeof(tm_buffer), "%Y-%m-%d %H:%M:%S", tm_info);
 
 	vty_out(vty, "PCEP counters since %s (%luh %lum %lus):\n", tm_buffer,
-		diff_time / 3600, (diff_time / 60) % 60, diff_time % 60);
+		(long)(diff_time / 3600), (long)((diff_time / 60) % 60), (long)(diff_time % 60));
```

Comment:
I'd prefer using more standard types and format specs - like "uint32_t" and "%u"